app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+
```

Comment:
It would be nice to use the `after_save` call back to call the `#update_post_counter` method every time a new post has been saved to the database. Please read more about database callbacks [here](https://guides.rubyonrails.org/active_record_callbacks.html).